common/app/conf/HealthCheck.scala

```diff
@@ -19,27 +18,21 @@ import scala.util.control.NonFatal
 
 sealed trait SingleHealthCheck {
   def path: String
-  def expires: HealthCheckExpiration
+  def expires: Option[Duration]
```

Comment:
Not related. Simplifying a bit the code. I guess I wrote that previously at a time I was quite excited about ADT :rofl: